tests/multitenant_tests/case_abc.py

```diff
@@ -0,0 +1,20 @@
+from abc import ABC, abstractmethod
+
+from fastapi import Response
+from fastapi.testclient import TestClient
+
+from mealie.repos.repository_factory import AllRepositories
+
+
+class ABCMultiTenanatTestCase(ABC):
```

Comment:
This should probably be `ABCMultiTenantTestCase`? 😅

Reply:
Ha! That would be better I think 👍